lithops/storage/backends/aws_s3/config.py

```diff
@@ -16,14 +16,18 @@
 
 
 import copy
+import logging
 
+logger = logging.getLogger(__name__)
 
-def load_config(config_data):
 
+def load_config(config_data):
     if 'aws' in config_data:
 
-        if not {'access_key_id', 'secret_access_key'}.issubset(set(config_data['aws'])):
-            raise Exception("'access_key_id' and 'secret_access_key' are mandatory under the 'aws' section of the configuration")
+        if "secret_access_key" in config_data["aws"] or "access_key_id" in config_data["aws"]:
+            logger.warning("Using 'secret_access_key' and 'access_key_id' in lithops configuration is not recommended "
+                           "- Use boto3 configuration file in ~/.aws or environment variables instead "
+                           "(https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html)")
 
```

Comment:
No need for any warning for now. We still have to decide whether we want to deprecate it or not

Reply:
I can't add comments in unmodifed code, but:
In lines 42-43: if in options 1 and 2 region is not mandatory, this should be fixed.
Now lithops supports the automatic creation of the storage bucket if it is not provided in the config. So lines 45-48 should be fixed.

Reply:
Yes, I think we still need the region name for [this](https://github.com/lithops-cloud/lithops/blob/35599ecd198132c56fbdeaca36eb12164768f876/lithops/serverless/backends/aws_lambda/aws_lambda.py#L83C9-L83C17) or [this](https://github.com/lithops-cloud/lithops/blob/35599ecd198132c56fbdeaca36eb12164768f876/lithops/serverless/backends/aws_lambda/aws_lambda.py#L638C45-L638C45). In any case, the region from the Lithops config would override the region stated in the aws config.

Reply:
> Now lithops supports the automatic creation of the storage bucket if it is not provided in the config. So lines 45-48 should be fixed.

Not sure how to proceed with this... With the SSO approach, we don't have a "fixed" key or ID to read from, contrary to the key pair approach.

Reply:
Is the S3 instance shared between all the users in the SSO approach? 
I wonder if we can simply use the `config_profile` name (or a hash of it) for the bucket name

Reply:
Multiple SSO users can access to a same account and share buckets, but each one will have a different profile name. Yes, using config profile is viable :+1: